spec/bambora/bank/adapters/payment_profile_response_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+module Bambora
+  module Bank
+    module Adapters
+      describe PaymentProfileResponse do
+        describe '#to_h' do
+          context 'a valid response' do
+            let(:json_response_body) { { someGelflings1: %w[rian deet brea] } }
+            let(:query_string_response_body) { URI.encode_www_form(json_response_body) }
+            let(:faraday_response) { Faraday::Response.new(body: query_string_response_body) }
+            subject { described_class.new(faraday_response).to_h }
+            it { is_expected.to eq some_gelflings_1: %w[rian deet brea] }
+          end
+
+          context 'an invalid response' do
```

Comment:
The new rubocop linting rules are always telling me to add `with` or `when` to a context description, could you check?

Reply:
@sweetyclem with the current `rubocop` version that is being used in this repository, we would have to manually add [`rubocop-rspec`](https://github.com/rubocop-hq/rubocop-rspec) to the dependencies to do spec linting.

This is something that is definitely worthwhile, to allow consistency on the spec writing. I like the guideline you mentioned!

Reply:
Ohh I didn't realize this was not in hm-proj!

Reply:
Oh, I like `when`. I'll add that.